backend/infrahub/core/schema/attribute_schema.py

```diff
@@ -195,19 +201,13 @@ class TextAttributeSchema(AttributeSchema):
     @model_validator(mode="after")
     def reconcile_parameters(self) -> Self:
         if self.regex != self.parameters.regex:
-            final_regex = self.parameters.regex or self.regex
-            if not final_regex:  # falsy parameters.regex override falsy regex
-                final_regex = self.parameters.regex
+            final_regex = self.parameters.regex if self.parameters.regex is not None else self.regex
```

Comment:
It looked buggy with falsy values, ie this test https://github.com/opsmill/infrahub/pull/6498/files#diff-4756160af30ba34332995a304a5e550284ac6f6bc2f98dd2b28c305aca218353R160 was failing because value set was `None` instead of 0